Source/AcceleratorLattice/AcceleratorLattice.cpp

```diff
@@ -88,7 +88,7 @@ AcceleratorLattice::InitElementFinder (int const lev, amrex::BoxArray const & ba
 }
 
 void
-AcceleratorLattice::UpdateElementFinder (int const lev)
```

Comment:
We could consider NOLINT here. Technically this could be a const function. However, it's only because `m_element_finder` is a `std::unique_ptr`. But that's an implementation detail. To the user, it actually makes more sense to not mark this as const.

Reply:
https://isocpp.org/wiki/faq/const-correctness#logical-vs-physical-const